package.json

```diff
@@ -121,6 +122,7 @@
     "eslint-plugin-react": "7.14.2",
     "exports-loader": "0.7.0",
     "file-loader": "4.1.0",
+    "full-icu": "1.3.0",
```

Comment:
What use are we giving to ICU?

Reply:
I asked about it above. It's used to provide internationalization data to Node when running tests since it doesn't include it by default

https://github.com/mattermost/mattermost-webapp/pull/3866#discussion_r339653703